hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -138,7 +138,7 @@
     # https://docs.openstack.org/glance/latest/admin/quotas.html
     /opt/stack/devstack/tools/upload_image.sh https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/cirros/2022-12-05/cirros-0.6.1-x86_64-disk.img
     /opt/stack/devstack/tools/upload_image.sh https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/ubuntu/2023-01-14/focal-server-cloudimg-amd64.img
-    /opt/stack/devstack/tools/upload_image.sh https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/flatcar/flatcar-stable-3510.2.4-kube-v1.27.2.img
+    /opt/stack/devstack/tools/upload_image.sh https://storage.googleapis.com/tormath1-flatcar/flatcar-stable-3602.2.0-kube-v1.27.2.img
```

Comment:
Is there any way in which we wouldn't have to switch between `artifacts.k8s-staging-capi-openstack.appspot.com/test` and `tormath1-flatcar` whenever a new flatcar image is being made available?

I appreciate that an automatic process would be best (cf. https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/1502), but maybe we can find a different solution in the interim?

Reply:
> Yes, it's not ideal and it creates bottleneck: one needs to generate the image with the image-builder and one with CAPO GCS Bucket write access needs to upload it.

Indeed, that's exactly what I was wondering we could achieve here: Find a person with the permissions above or grant them.

> One different approach would be to remove the image-builder dependency and to use plain Flatcar image: [...] In the Flatcar template, using Ignition, we could pull Kubernetes dependencies using [systemd-sysext](https://www.flatcar.org/docs/latest/container-runtimes/getting-started-with-kubernetes/#setup-the-control-plane):

I am *very* much looking forward to utilising that functionality in all my clusters and, quite likely, in the context of tests here. How far has that feature progressed by now? My impression was that it is still quite experimental. But, if we can make it work, I'd very much like that. Until then, my preference would be for tests to mirror the setup that most users would use.

On that note: Please give me a shout if you need any help in getting the templates ready.

Reply:
I still think the ideal situation here is to generate them directly in the CI. Incidentally, the 'OpenStack' packer builder is vastly quicker and more reliable than the qemu builder. The only problem with it is you need an OpenStack to run it against. That's hurting us.